certbot/display/util.py

```diff
@@ -23,9 +23,6 @@
 CANCEL = "cancel"
 """Display exit code for a user canceling the display."""
 
-HELP = "help"
```

Comment:
Ah, good catch! I saw your comment and was thinking about it in the context of #3915, but not #4795.

I'll revisit this with that in mind.